### PR TITLE
support to convert bit to boolean

### PR DESCRIPTION
**Pull request by bharadwaj-aditya**
Added support to convert from bit(1) to boolean.
The default mapping is that bit(1) will migrate to boolean.

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

**Comment by asthamohta** [comment]
Do we add a similar issue for bit(1)?
https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/2da4d00ef7dbe37a2618dfd04eefb98bd8cf5618/webv2/web.go#L3093

File: sources/mysql/data.go [hunk 1]
```diff
@@ -137,9 +137,22 @@ func convScalar(conv *internal.Conv, spannerType ddl.Type, srcTypeName string, T
...
+func convBool(conv *internal.Conv, spannerType ddl.Type, srcTypeName string, val string) (bool, error) {
 	b, err := strconv.ParseBool(val)
 	if err != nil {
+
+		if srcTypeName == "bit" {
+			// To handle scenarios where bit is used to store boolean
+			// zero treated as false
+			// one treated as true
+			switch val {
+			case "\x00":
```
Comment (Deep1998) [inline sources/mysql/data.go:149]: Will this byte string be guaranteed to be 2 digit - either "\x00" or "\x01"? Can source not be "\x1" or "\x001" etc?

Should we instead parse to number if "starts with \x" and compare to 1 or 0?

Comment (bharadwaj-aditya) [reply sources/mysql/data.go:149]: This is how 0 and 1 are represented in bytes in go

Comment (Deep1998) [reply sources/mysql/data.go:149]: I see, bit(1) data is always read as these 2 string we should be good

**Comment by Deep1998** [comment]
LGTM

**Review by Deep1998 — APPROVED** [review]

**Pull request merged**